analyzers/tests/SonarAnalyzer.Test/TestCases/AspNet/RouteTemplateShouldNotStartWithSlash.AspNetCore.vb

```diff
@@ -312,3 +312,12 @@ Public Class ControllerRequirementsInfluenceActionsCheck
     End Class
 
 End Class
+
+' https://github.com/SonarSource/sonar-dotnet/issues/9002
+Public Class Repro_9002                     ' Noncompliant
+    Inherits Controller
+    <Route("~/B")>                          ' Secondary
```

Comment:
It would be nice to place an empty line between end of class declaration and before method
```suggestion
    Inherits Controller
    
    <Route("~/B")>                          ' Secondary
```